builtin/mainmenu/tab_online.lua

```diff
@@ -111,13 +113,44 @@ local function get_formspec(tabview, name, tabdata)
 		if gamedata.fav then
 			retval = retval .. "tooltip[btn_delete_favorite;" .. fgettext("Remove favorite") .. "]"
 			retval = retval .. "style[btn_delete_favorite;padding=6]"
-			retval = retval .. "image_button[5,1.3;0.5,0.5;" .. core.formspec_escape(defaulttexturedir ..
+			retval = retval .. "image_button[4.5,1.3;0.5,0.5;" .. core.formspec_escape(defaulttexturedir ..
 				"server_favorite_delete.png") .. ";btn_delete_favorite;]"
 		end
 		if gamedata.serverdescription then
 			retval = retval .. "textarea[0.25,1.85;5.25,2.7;;;" ..
 				core.formspec_escape(gamedata.serverdescription) .. "]"
 		end
+
+		local function shortenTable(clients_list)
```

Comment:
Should be `shorten_table` according to our naming conventions, see https://dev.minetest.net/Lua_code_style_guidelines

(also, this could be inlined at the only place where it is used. whether that is more readable or not may be debatable, but at least the `math.min` becomes obsolete then: You just have to pick the first 5 because you know that there are `>= 5`. actually, you can always pick the first 5 because `nil`s in Lua are funny but that wouldn't be very clean)